prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -273,17 +280,17 @@ def setup_session(
                     return KubernetesSession(
                         api_client=client.ApiClient(), context=context
                     )
-            if context:
-                contexts = config.list_kube_config_contexts(
-                    config_file=kubeconfig_file
-                )[0]
-                for context_item in contexts:
-                    if context_item["name"] == context:
-                        context = context_item
-            else:
-                context = config.list_kube_config_contexts(config_file=kubeconfig_file)[
-                    1
-                ]
+                if context:
+                    contexts = config.list_kube_config_contexts(
+                        config_file=kubeconfig_file
+                    )[0]
+                    for context_item in contexts:
+                        if context_item["name"] == context:
+                            context = context_item
+                else:
+                    context = config.list_kube_config_contexts(
+                        config_file=kubeconfig_file
+                    )[1]
```

Comment:
Why are you getting the `[1]`?

Reply:
Because that is the active context, that was not changed.

Reply:
If the `active` is the second element, what is in the first one?

Reply:
The first one is a list of all the contexts, the second one is an object of the active context.